### DISCUSSION TITLE
500 Error for Basic GPT Tutorial {"msg":"InvalidWorkerCreation: worker boot error"}%

### DISCUSSION THREAD

**Discussion by bnjreece** [supabase/supabase#15874] [Questions]
HI, I'm trying to follow this basic template here: https://github.com/supabase/supabase/tree/master/examples/edge-functions/supabase/functions/openai and 
here: https://supabase.com/docs/guides/ai/examples/openai

I cannot get a response from the API, I get the above error. I've tried restarting computer/supabase/docker/starting from scratch, making sure VS is configured correctly from this video: https://www.youtube.com/watch?v=29p8kIqyU_Y&t=638s

But no matter what I do, I get the 500 error 😦
![image](https://github.com/supabase/supabase/assets/35317721/a8f49dbe-2861-41ae-bc53-99238bfc64d7)
![image](https://github.com/supabase/supabase/assets/35317721/f6d4ebb1-f3fe-4bbe-8b50-fca6a674948f)
![image](https://github.com/supabase/supabase/assets/35317721/d774734f-5037-41b8-b5d3-668ad3ef3366)

**Comment by GaryAustin1** [comment]
You need to either include those imports in the import_map.json file or put in the full url for the import.
https://github.com/supabase/supabase/blob/master/examples/edge-functions/supabase/functions/import_map.json
